### DISCUSSION TITLE
timed traffic lights with order

### DISCUSSION THREAD

**Discussion by xxjoe2** [slyh/Cities2-TrafficLightsEnhancement#32] [Ideas]
hi

wish that you would consider adding timed traffic lights as what in cs1 (mod) to control the time allowed for each side, since most of the juctions are not busy at all sides with one or two having very few traffic

also it would be great to add a feature to let the user to choose the order which side will go first as the old mod did. eg north, south, east, north then
west in loop, and advanced mode like north+south go straight with left turn allowed (left side drive), then east followed by west etc with time outs

thanks a lot

**Comment by slyh** [comment]
At this stage I'm not certain if such functionality will be implemented.
